src/main/scala/fr/polytechnique/cmap/cnam/etl/extractors/ngapacts/McoCeNgapActExtractor.scala

```diff
@@ -0,0 +1,74 @@
+package fr.polytechnique.cmap.cnam.etl.extractors.ngapacts
+
+import fr.polytechnique.cmap.cnam.etl.events.{Event, EventBuilder, McoCeFbstcNgapAct, McoCeFcstcNgapAct, NgapAct}
+import fr.polytechnique.cmap.cnam.etl.extractors.mcoCe.McoCeExtractor
+import fr.polytechnique.cmap.cnam.etl.sources.Sources
+import org.apache.spark.sql.{DataFrame, Dataset, Row}
+
+import scala.reflect.runtime.universe._
+import scala.util.Try
+
+trait McoCeNgapActExtractor extends McoCeExtractor[NgapAct] {
+  val ngapActsConfig: NgapActConfig
+  val keyLetterColumn: String
+  val coeffColumn: String
+
+  val columnName: String = keyLetterColumn
+
+  override def isInStudy(codes: Set[String])(row: Row): Boolean = {
+    ngapActsConfig.pmsiIsInCategories(
```

Comment:
This a bit weird. I don't understand why a config class is responsible to check if a given row is in the study. This should be the responsability of the Extractor not the Config.

Reply:
I put the function which check if a given row is in the study within the extractor